Sources/Core/Utilities/Logger/Logger.swift

```diff
@@ -64,25 +64,47 @@ public extension Logger {
   static func error(_ message: String, params: Parameters? = nil, file: String = #file, function: String = #function, line: Int = #line) {
     flush(.error, message: message, params: params, file: file, function: function, line: line)
   }
-  
-  /// Log crash
-  static func crash(params: Parameters, file: String = #file, function: String = #function, line: Int = #line) {
-    flush(.error, message: "app_crash", params: params, file: file, function: function, line: line)
-  }
 }
 
 // MARK: - Private Methods
 private extension Logger {
   static func flush(_ level: LogLevel, message: String, params: Parameters? = nil, file: String, function: String, line: Int) {
     guard shared.logLevel.rawValue >= level.rawValue else { return }
     
-    let fileName = URL(fileURLWithPath: file).lastPathComponent.components(separatedBy: ".").first ?? ""
-    var messagePrint = "\(level.string): \(fileName).\(function)[\(line)]: \(message)"
-    if let params = params, !params.isEmpty {
-      messagePrint += ", params: \(params)"
+    let fileName = URL(string: file)?.lastPathComponent ?? ""
```

Comment:
I think we still should use `URL(fileURLWithPath: file)` , to ensure the path is correctly interpreted as a file path

Reply:
Good point 👌